src/components/layout/layout.tsx

```diff
@@ -1,6 +1,6 @@
 import { ReactNode } from 'react';
-import Footer from './footer/footer';
-import Header from './header/header';
+import Footer from './Footer/Footer';
```

Comment:
te importy raczej powinny zostać z małej bo takie są foldery i ts sie pruje jak jest z dużej. 🤔

Reply:
I did not have problem with this, even after changing to lower case 🙄...

Reply:
🤔🤔🤔